docs/views/expander.md

```diff
@@ -0,0 +1,335 @@
+---
+title: "Xamarin.CommunityToolkit Expander"
```

Comment:
Should we just name this `Expander`? We're already in the Xamarin.CommunityToolkit docs subsection. Feels like overly repeating that is a bit overkill.

Reply:
No. The metadata is used for many purposes including SEO, analytics, and reporting. It's best to preface each control with Xamarin.CommunityToolkit. Aside from anything else, the title metadata isn't displayed on screen.

Reply:
Thanks David! Would you suggest keeping the library name in the title and not in the title that is actually shown on the page? Or have it there as well? Just checked and that is how it is for the [current Expander in Forms](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/expander) as well

Reply:
Most users will land on a doc from a search engine, and including the product name in the H1 (the title shown on the page) will let them know exactly what product the doc is for.

The long version of what I've just said can be found at:

Tips for writing titles - https://review.docs.microsoft.com/help/contribute/contribute-how-to-write-good-titles?branch=master
Tips for writing H1s - https://review.docs.microsoft.com/help/contribute/contribute-how-to-write-good-h1?branch=master